tests/e2e/index.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from "@playwright/test";
+
+test.describe("Homepage (_index route)", () => {
+  test("has title, shows headline and introduction text", async ({ page }) => {
+    await page.goto("/");
+    await expect(page).toHaveTitle("Kommunikationsplattform | Justiz-Services");
+    await expect(page.locator("text=Kommunikationsplattform")).toBeVisible();
+    await expect(
+      page.locator(
+        "text=Willkommen auf der Pilotplattform für den digitalen Austausch zwischen Gerichten und Verfahrensbeteiligten.",
+      ),
+    ).toBeVisible();
+  });
+
+  test('redirects to beA-Portal when using the "Anmeldung Anwaltschaft" login option', async ({
+    page,
+  }) => {
+    // triple timeout, see: https://playwright.dev/docs/api/class-test#test-slow
+    test.slow();
+    await page.goto("/");
+    await page
+      .getByRole("link", {
+        name: "Anmeldung Anwaltschaft",
+      })
+      .click();
+    await page.waitForURL((url) => url.toString().includes("schulung"));
+
+    // expected URL partial after clicking the beA-Portal login link
+    const expectedUrl =
+      /schulung\.bea-brak\.de\/auth\/realms\/brak\/protocol\/openid-connect\/auth/;
+
+    await expect(page).toHaveURL(expectedUrl);
+    await expect(page.locator("h1")).toHaveText("Anmeldung");
+  });
+
+  test("important footer links are visible and clickable.", async ({
+    page,
+  }) => {
+    await page.goto("/");
+    await expect(page.getByRole("link", { name: "Datenschutz" })).toBeVisible();
+    await page.getByRole("link", { name: "Datenschutz" }).click();
+    await expect(page).toHaveURL("/datenschutz");
+    await page.goto("/");
+    await expect(page.getByRole("link", { name: "Impressum" })).toBeVisible();
+    await page.getByRole("link", { name: "Impressum" }).click();
+    await expect(page).toHaveURL("/impressum");
+  });
+
+  test(`shows success alert box for status=logged-out URL param (will be shown after using the logout button)`, async ({
+    page,
+  }) => {
+    await page.goto("/?status=logged-out");
+    await expect(page.getByRole("alert")).toContainText(
+      "Erfolgreich abgemeldet",
+    );
+  });
+
+  test(`shows warning alert box for status=auto-logged-out URL param (will be shown after 60 min. of inactivity to our logged in users)`, async ({
+    page,
+  }) => {
+    await page.goto("/?status=auto-logged-out");
+    await expect(page.getByRole("alert")).toContainText(
+      "Automatisch abgemeldet",
+    );
+  });
+});
```

Comment:
…and then add another describe block to iterate over the links and test them one-by-one:

```suggestion
  test.describe("Footer links navigation", () => {
    test.beforeEach(async ({ page }) => {
      await page.goto("/");
    });
    test("should navigate to correct pages when clicking footer links", async ({
      page,
    }) => {
      await page.goto("/");
      for (const { label, url, h1 } of links) {
        await page.getByRole("link", { name: label }).click();
        await expect(page).toHaveURL(url);
        await expect(page.locator("h1")).toHaveText(h1);
      }
    });
  });
});
```